aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+///
+///         Do you want to submit a transaction for a range of [52300 - 78400] Octas at a gas unit price of 100 Octas? [yes/no] >
+///         yes
+///         {
+///           "Result": {
+///             "transaction_hash": "0xd3f7bab04e18f4e631f40e669bf1e3224d8c08318da509aeedf1806cc0cd6cdd",
+///             "gas_used": 523,
+///             "gas_unit_price": 100,
+///             "sender": "c9f5ffb4fa164729e28733d5878ee4e90604478640dfb5ccb477452c0fe1cd79",
+///             "sequence_number": 18,
+///             "success": true,
+///             "timestamp_us": 1683077797249592,
+///             "version": 3652742,
+///             "vm_status": "Executed successfully"
+///           }
+///         }
+///
+///     View the transaction on the explorer here:
+///     https://explorer.aptoslabs.com/txn/0xd3f7bab04e18f4e631f40e669bf1e3224d8c08318da509aeedf1806cc0cd6cdd?network=devnet
+module mint_nft_v2::create_nft {
+    use std::bcs;
+    use std::error;
+    use std::signer;
+    use std::object;
+    use std::string::{Self, String};
+    use aptos_framework::account;
+
+    use aptos_token_objects::aptos_token::{Self, AptosToken};
+    use aptos_token_objects::collection;
+
+    // This struct stores an NFT collection's relevant information
+    struct ModuleData has key {
+        collection_name: String,
+        creator: address,
+        token_description: String,
+        token_name: String,
+        token_uri: String,
+        property_keys: vector<String>,
+        property_types: vector<String>,
+        property_values: vector<vector<u8>>,
+        collection_object_address: address,
+    }
+    /// Action not authorized because the signer is not the admin of this module
+    const ENOT_AUTHORIZED: u64 = 1;
+
+    /// `init_module` is automatically called when publishing the module.
+    /// In this function, we create an example NFT collection and an example token.
+    fun init_module(source_account: &signer) {
+        let collection_name = string::utf8(b"Collection name");
+        let description = string::utf8(b"Description");
+        let collection_uri = string::utf8(b"Collection uri");
+        let token_name = string::utf8(b"Token name");
+        let token_uri = string::utf8(b"Token uri");
+        let maximum_supply = 1000;
+
+        aptos_token::create_collection(
+            source_account,
+            description,
+            maximum_supply,
+            collection_name,
+            collection_uri,
+            false, // mutable_description
+            false, // mutable_royalty
+            false, // mutable_uri
+            false, // mutable_token_description
+            false, // mutable_token_name
+            true, // mutable_token_properties
+            false, // mutable_token_uri
+            false, // tokens_burnable_by_creator
+            false, // tokens_freezable_by_creator
+            5, // royalty_numerator
```

Comment:
Let's add a quick comment here to explain that this is 5% royalty - 5% of all transfers should be sent to the creator.